interactions/models/internal/context.py

```diff
@@ -172,8 +169,10 @@ class BaseContext(metaclass=abc.ABCMeta):
     guild_id: typing.Optional[Snowflake]
     """The id of the guild this context was invoked in, if any."""
 
-    def __init__(self, client: "interactions.Client") -> None:
-        self.client = client
+    def __init__(self, client: ClientT) -> None:
+        self.client: ClientT = client
+        """The client that created this context."""
```

Comment:
If you're wondering about this - I had a hard time convincing typehinting that `self.client` wasn't `Unknown` without resorting to this.